fuels-rs/src/contract.rs

```diff
@@ -294,8 +303,8 @@ where
     /// `Result<bool, Error>`. Also works for structs! If your method
     /// returns `MyStruct`, `MyStruct` will be generated through the `abigen!()`
     /// and this will return `Result<MyStruct, Error>`.
-    pub async fn call(self) -> Result<D, Error> {
-        let receipts = Contract::call(
+    pub async fn call(self) -> Result<CallResponse<D>, Error> {
```

Comment:
Do you mind updating this function's doc-level comments?

Reply:
sure, forgot